website/docs/r/cognito_user_pool_domain.markdown

```diff
@@ -34,9 +34,23 @@ resource "aws_cognito_user_pool_domain" "main" {
 resource "aws_cognito_user_pool" "example" {
   name = "example-pool"
 }
-```
 
+data "aws_route53_zone" "example" {
+  name = "example.com"
+}
 
+resource "aws_route53_record" "auth-cognito-A" {
+  name    = "${aws_cognito_user_pool_domain.main.domain}"
+  type    = "A"
+  zone_id = "${aws_route53_zone.example.zone_id}"
```

Comment:
I think this needs to be `data.aws_route53_zone.example.zone_id`, or at least it needed to be for me when using terraform 0.12